src/app/plugins/cat.plugin.ts

```diff
@@ -50,7 +45,10 @@ export class CatPlugin extends Plugin {
 
     if (args[0].includes('breed')) {
       //Simply return the list of supported breeds
-      message.reply(this._embedBreeds);
+      if (!this._embedBreeds) {
+        await this._getListEmbed();
+      }
+      message.reply(this._embedBreeds || 'Failed to load breeds.');
```

Comment:
nit: do the `if` check in the `_getListEmbed` func so that the caller doesnt have to think about when to call it